livre/eq/render/RayCastRenderer.cpp

```diff
@@ -430,9 +429,13 @@ struct RayCastRenderer::Impl
 
     void onFrameRender( const NodeIds& bricks )
     {
+        createAndFillVertexBuffer( bricks );
         size_t index = 0;
         for( const NodeId& brick: bricks )
             renderBrick( brick, index++ );
+
+        glDeleteBuffers( 1, &_posVBO );
+        glFlush();
```

Comment:
A lonely function returning (or worse creating and not returning) resources which have to be freed is always bad design. Sure the delete is only a one-liner, so?

In the end I'm fine if at least the fbo is returned to the caller and not stored as a member.